.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: go
+
+os:
+  - linux
+  - osx
```

Comment:
osx isn't necessary as there is no platform dependent code in this repo, we are trying to stop using it as there is constant queue for OSX workers in Travis (next unspoken rule).